tests/tests.js

```diff
@@ -158,6 +146,59 @@ describe('Discord Tests', function() {
     });
 
     /**
+     * Test that the database is updated correctly
+     */
+    describe('Database Tests', function() {
+        describe('Ping', function() {
+            it('Ping events are recorded', function(done) {
+                models.Ping.count().then(function(countBeforePing) {
+                    assert.equal(countBeforePing, 0);
+
+                    // Send a ping
+                    sendPost(testUtils.nonRecordedFixturesDir + '1', function(error, response, body) {
+                        models.Ping.count().then(function(countAfterPing) {
+                            assert.equal(countAfterPing, 1);
+
+                            // Send another ping
+                            sendPost(testUtils.nonRecordedFixturesDir + '1', function(error, response, body) {
+                                models.Ping.count().then(function(countAfterPing) {
+                                    assert.equal(countAfterPing, 2);
+                                    done();
+                                });
+                            });
+                        });
+                    });
+                });
+            });
```

Comment:
We definitely have callback hell here. I guess I wasn't worried about that in tests. We could use `q` if we want to flatten things out. What do you think @darkwing?